delivery_carrier_b2c/README.rst

```diff
@@ -0,0 +1,36 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+ :alt: License: AGPL-3
+
+Delivery Carrier Business To Customer
+=====================================
+
+This module was written for add specific fields to partner for delivery purpose
```

Comment:
'This module was written to add specific fields to partners...', or maybe better: 'This module adds specific fields to partners...'

Can you add some information about which fields are added and how they influence the business logic?